src/main/scala-2.10/com/mpc/scalats/core/ScalaParser.scala

```diff
@@ -225,6 +227,10 @@ final class ScalaParser(logger: Logger) {
   @inline private def isCaseClass(scalaType: Type): Boolean =
     scalaType.typeSymbol.isClass && scalaType.typeSymbol.asClass.isCaseClass
 
+  @inline private def isCaseClassAnyVal(scalaType: Type) = {
```

Comment:
If you are thinking of value class, that's not case class (about naming)

Reply:
Ok I will change that